controllers/rollingupgrade_controller.go

```diff
@@ -375,7 +375,8 @@ func (r *RollingUpgradeReconciler) SetStandby(ruObj *upgrademgrv1alpha1.RollingU
 
 	instanceState, err := getGroupInstanceState(asg, instanceID)
 	if err != nil {
-		return err
+		r.info(ruObj, fmt.Sprintf("WARNING: %v", err))
```

Comment:
Please file a bug for creating the `warn()` method in `RollingUpgradeReconciler`. This mechanism of using an info level message with `WARNING` as the text in it is not ideal.